doc/cookbook/source/examples/binary_classifier/averaged_perceptron.rst

```diff
@@ -0,0 +1,61 @@
+===================
+Averaged Perceptron
+===================
+
+The Averaged Perceptron is an online binary classifier which is used to determine whether an input,
```

Comment:
Don't explain what classification is ;) This kind of stuff is more for the notebooks / longer examples. This is supposed to be a minimal.
The averaged Perceptron is an online binary classifier.